kanidm_book/src/installing_the_server.md

```diff
@@ -109,6 +109,11 @@ in place you can use a shell container that mounts the volume such as:
 
     docker run --rm -i -t -v kanidmd:/data -v /my/host/path/work:/work opensuse/leap:latest cp /work/* /data/
 
+If the above command says: `cp: cannot stat /work/*: No such file or directory`, you can instead run the following:
```

Comment:
The original command in line 110 probably should be updated so it works regardless of shell - I think quoting it would fix it?

Reply:
Yes, it needs quoting I think. Because the * is probably expanded to early.